### PR TITLE
02_YouSayLowISayCode

### PR DESCRIPTION
**Pull request by aprildunnam**

### CODE REVIEW

File: website/blog/2023-lcf/2023-02-01/02_YouSayLowISayCode.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+![Fusion Development Process](./02_FusionDevProcess.png)
+
+## What is "low code"?
+
+The term "low code" can mean different things to different people when they first hear it. What we mean when we talk about low code is that with tools like Power Apps, you only need to write a small amount of code to get results that would normally take several more lines of code in a traditional programming language.
+
+![What is Low Code](./02_WhatIsLowCode.png)
+
+In canvas-based Power Apps, the low code scripting language used is called [Power Fx](https://aka.ms/LearnPowerFx). This language is based on the Microsoft Excel formula language which meets many business technologists where they are by leveraging the knowledge they already have.
```
Comment (codemillmatt) [inline website/blog/2023-lcf/2023-02-01/02_YouSayLowISayCode.md:27]: Would it make sense to put this section above the previous? So we'd explain right away what low code is?

- You say low, I say code
- What is low code
- Low code + full code = no cliffs

File: website/blog/2023-lcf/2023-02-01/02_YouSayLowISayCode.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+![Power Platform Pro Dev extensions](./02_Extend.png)
+
+## What's next?
+
+In the next series of posts you'll learn more about some of these core concepts and extension points. 
+
+If all of this sounds intriguing and you want to get started on your low code development journey, here are some actions you can take:
+
+* 1️⃣ [**Sign up**](https://aka.ms/lowcode-february/devplan) for the free Power Apps Developer Plan.
+* 2️⃣ [**Enroll**](https://aka.ms/lowcode-february/challenge) in the Cloud Skills Challenge and start climbing that leaderboard!
```
Comment (codemillmatt) [inline website/blog/2023-lcf/2023-02-01/02_YouSayLowISayCode.md:61]: Maybe here: 

> Enroll in the Cloud Skills Challenge to learn more about low-code and fusion team fundamentals and start climbing that leaderboard!

**Review by codemillmatt — APPROVED** [review]
I think everything is good as-is. I left some comments - up to you if you want to incorporate them or not.

File: website/blog/2023-lcf/2023-02-01/02_YouSayLowISayCode.md (hunk 1 above)
Comment (aprildunnam) [reply website/blog/2023-lcf/2023-02-01/02_YouSayLowISayCode.md:19]: Great suggestion Matt. I moved it up in the blog

**Review by aprildunnam — COMMENTED** [review]
updates made based on review

**Pull request merged**